articles/integrations/azure-api-management/configure-azure.md

```diff
@@ -0,0 +1,172 @@
+
+## Configure Azure
+
+::: note
+To complete this, tutorial, you will need to have an account that grants you access to Microsoft's [Azure Portal](https://portal.azure.com).
+:::
+
+In this section, you'll begin by creating an API management instance, as well as the API to manage your cookies application.
```

Comment:
I don't know the first thing about this integration, so maybe it's just me, but I'd want more context on this intro. What is this `cookies application`?